lib/health_checking/health_checks/board/issue_blocked_label_check.rb

```diff
@@ -0,0 +1,31 @@
+module HealthChecking
+  module HealthChecks
+    module Board
+      class IssueBlockedLabelCheck
+        include HealthCheck
+        name 'issue_blocked_label_check'
+        weight :warning
+        authorization :collaborator
+        passed "Issue blocked label is present"
+        failed "Issue blocked label is not present"
```

Comment:
Maybe `"Blocked" label is not present`? "Issue" seems redundant, and maybe more explicitly mentioning the expected label name would nudge people toward renaming their own? Or we could add some sort of tooltip to give a help bubble with more explanation.